src/bt-hf-io.c

```diff
@@ -217,12 +257,20 @@ dial_call_cmd_cb(char* number)
 
   len = strlen(number) + 1;
 
-  wbuf = create_pdu_wbuf(len, /* phone number */
+  wbuf = create_pdu_wbuf(6 + /* address (BlueZ 5.25) */
+                         len, /* phone number */
                          0, NULL);
   if (!wbuf)
     return;
 
   init_pdu(&wbuf->buf.pdu, SERVICE_BT_HF, OPCODE_DIAL_CALL_CMD_NTF);
+#if ANDROID_VERSION >= 21
+  /* It's a little weird to put bd_addr before any other parameters, but this
+   * order is defined in BlueZ 5.25 anyway.
```

Comment:
Drop this comment. I like to rant about Bluedroid as much as anyone else, ;) but this comment has no value or information.